src/GlobalPayments.Api/Terminals/TerminalUtilities.cs

```diff
@@ -118,7 +148,8 @@ public class TerminalUtilities {
             Bitmap bmp = new Bitmap(width, 100);
 
             var gfx = Graphics.FromImage(bmp);
-            gfx.Clear(Color.White);
+            // TODO: Remove color from Utilities
+            //gfx.Clear(Color.White);

```

Comment:
Why?

Reply:
Apologies for this, I forgot to uncomment this line. But the reason why I commented on this line is because I've experienced build error. Anyways I've already fixed it and forget to uncomment in again.

Reply:
![image](https://user-images.githubusercontent.com/61694338/86433975-c1746f00-bd2e-11ea-819f-5726bcdf7dd1.png)
Is it okay to replace the CoreCompat.System.Drawing package into System.Drawing.Common? Cause this cause a build error for Color enum.